Binary files linux-3.9.2/drivers/mtd/nand/s3c_nand_mlc.fo and drb-linux-3.9.2/drivers/mtd/nand/s3c_nand_mlc.fo differ
diff -ruN linux-3.9.2/Makefile drb-linux-3.9.2/Makefile
--- linux-3.9.2/Makefile	2013-05-11 16:19:28.000000000 +0200
+++ drb-linux-3.9.2/Makefile	2013-05-13 15:17:00.290798086 +0200
@@ -192,7 +192,7 @@
 # "make" in the configured kernel build directory always uses that.
 # Default value for CROSS_COMPILE is not to prefix executables
 # Note: Some architectures assign CROSS_COMPILE in their arch/*/Makefile
-ARCH		?= $(SUBARCH)
+ARCH		?= arm
 CROSS_COMPILE	?= $(CONFIG_CROSS_COMPILE:"%"=%)
 
 # Architecture as present in compile.h
